packages/quick_blue/windows/quick_blue_plugin.cpp

```diff
@@ -141,15 +141,19 @@ class QuickBluePlugin : public flutter::Plugin, public flutter::StreamHandler<En
   std::unique_ptr<flutter::BasicMessageChannel<EncodableValue>> message_connector_;
 
   std::unique_ptr<flutter::EventSink<EncodableValue>> scan_result_sink_;
+  std::unique_ptr<flutter::EventSink<EncodableValue>> availability_change_sink_;
 
   Radio bluetoothRadio{ nullptr };
+  RadioState oldBluetoothRadioState = RadioState::Unknown;
 
   BluetoothLEAdvertisementWatcher bluetoothLEWatcher{ nullptr };
   winrt::event_token bluetoothLEWatcherReceivedToken;
   void BluetoothLEWatcher_Received(BluetoothLEAdvertisementWatcher sender, BluetoothLEAdvertisementReceivedEventArgs args);
   winrt::fire_and_forget SendScanResultAsync(BluetoothLEAdvertisementReceivedEventArgs args);
 
   std::map<uint64_t, std::unique_ptr<BluetoothDeviceAgent>> connectedDevices{};
+  void Radio_StateChanged(Radio& radio);
```

Comment:
Move `Radio_StateChanged` below to `bluetoothRadio`

Reply:
Refactor `void Radio_StateChanged(Radio& radio);` to `void Radiao_StateChanged(Radio sender, IInspectable args);`